zokrates_cli/examples/compile_errors/ambiguous_generic_call.zok

```diff
@@ -0,0 +1,14 @@
+// this should not compile, as A == B
+
+const u32 A = 2
```

Comment:
judging by the comment above, shouldn't `A` and `B` have the same value here?

Reply:
yesss